airflow/example_dags/example_params_ui_tutorial.py

```diff
@@ -278,13 +276,8 @@
     },
 ) as dag:
 
-    @task(task_id="show_params")
-    def show_params(**kwargs) -> None:
-        ti: TaskInstance = kwargs["ti"]
-        dag_run: DagRun = ti.dag_run
-        if not dag_run.conf:
-            print("Uups, no parameters supplied as DagRun.conf, was the trigger w/o form?")
-            raise AirflowSkipException("No DagRun.conf parameters supplied.")
-        print(f"This DAG was triggered with the following parameters:\n{json.dumps(dag_run.conf, indent=4)}")
+    @task
+    def show_params(params: ParamsDict = dag.params) -> None:
+        print(f"This DAG was triggered with the following parameters:\n\n{json.dumps(params, indent=4)}\n")
```

Comment:
Is this change necessary? Can't we print the output of `dag_run.conf`?

Reply:
This example is about params.
While taking a look (again) on it I realized that it was using `dag_run.conf` previously. From what I saw how people use `dag_run.conf` it is actually in most cases better to consistently use `params` - and to make the tutorial consistent I adjusted it here.
Difference is: Even if scheduled or transmitted via API not passing conf, `params`is always there. `dag_run-conf` can be empty and you need to implement a fallback in all cases. This is just confusing.

Reply:
In my understanding, the purpose of the task was to show how you can get information from dag_run.conf and possibly use it in the dag. What do you think about showing both examples? No strong opinion though

Reply:
I'd propose to leave it rather with `params` as this tutorial DAG is for `params` actually. Users learning Airflow via this tutorial might be confused with `dag_run.conf` and if users have doubts then I'd propose to user `params` - because they will have a form and also be validated and have defaults as fallback - so generate less problems.